code-studio/src/js/initApp/initApp.js

```diff
@@ -93,6 +95,20 @@ window.apps = {
         var lastServerResponse = reporting.getLastServerResponse();
         if (lastServerResponse.videoInfo) {
           showVideoDialog(lastServerResponse.videoInfo);
+        } else if (lastServerResponse.endOfStageExperience) {
+          var body = document.createElement('div');
+          var props = {
+            stageName: `Stage ${lastServerResponse.previousStageInfo.position}: ${lastServerResponse.previousStageInfo.name}`,
+            onContinue: () => { dialog.hide(); },
+            i18n: window.dashboard.i18n
+          };
+          ReactDOM.render(<PlayZone {...props} />, body);
```

Comment:
Rather than pre-declaring a props object and then using spread notation, you could just pass in props directly
```
ReactDOM.render(
  <PlayZone
    stageName={...}
    onContinue:={() => dialog.hide() }
    i18n={window.dashboard.i18n}/>,
 body
);